execution/executor/src/logging.rs

```diff
@@ -1,27 +1,53 @@
 // Copyright (c) The Libra Core Contributors
 // SPDX-License-Identifier: Apache-2.0
 
-use libra_logger::StructuredLogEntry;
+use libra_crypto::HashValue;
+use libra_logger::Schema;
+use serde::Serialize;
 
-pub fn executor_log(entry: LogEntry) -> StructuredLogEntry {
-    StructuredLogEntry::new_named("executor", entry.as_str())
+#[derive(Schema)]
+pub struct LogSchema {
+    name: LogEntry,
+    block_id: Option<HashValue>,
+    root_block_id: Option<HashValue>,
+    original_reconfiguration_block_id: Option<HashValue>,
+    num: Option<u64>,
+    local_synced_version: Option<u64>,
+    first_version_in_request: Option<Option<u64>>,
```

Comment:
Umm? Why not just Option :)

Reply:
Idk that's just the type that was being logged lol!

Reply:
why not just use an option here though?

Reply:
Option<Option<_>> makes little sense but keeps consistency.